types/api/device.d.ts

```diff
@@ -51,10 +51,25 @@ export interface ApiDeviceLocationFixup {
   location?: LatLng; // Supply a location to map to the station
 }
 
+type SettingsBase = {
+  updated: IsoFormattedDateString;
+};
+
+export type ThermalRecordingSettings = {
+  useLowPowerMode: boolean;
+} & SettingsBase;
+
+export type WindowsSettings = {
+  startRecording: string;
+  stopRecording: string;
+  powerOn: string;
```

Comment:
`powerOn` and `powerOff` fields are ignored in DOC AI cameras, so not sure if we should include these at all.  I don't know that we'll ever go back to using them.